### PR TITLE
Issue 28. Add SOLR config in the Ngnix branch.

### PR DESCRIPTION
**Pull request by mtem27**
Issue #28 SOLR integration.

### CODE REVIEW

**Comment by Luukyb** [comment]
Thanks @mtem27, could you remove the .DS_Store file? Thanks

**Comment by Luukyb** [comment]
Hi @mtem27 ,
Could you port the same changes on another PR for the apache branch?

**Comment by Luukyb** [comment]
Hi @mtem27, did you see my previous comment? Also it looks like there is a conflict on the README file. Thanks!

**Comment by mtem27** [comment]
Hi @Luukyb,

I tested this again after updating SOLR version to latest and encountered the below error, reported in Issue #64 :
image

I added the suggested fix and the build was successful.

SOLR also works fine in my local, see below: 

image

Kindy review again.

Thanks.

Tere

File: README.md [hunk 1]
```diff
@@ -147,3 +147,7 @@ To access PhpMyAdmin, run `docker ps` to get its dynamic port.
 * Username: root
 * Password: root
 
+### SOLR Integration
+To enable SOLR, uncomment corresponding lines from docker-compose.yml file.
+Current version is 7.1.0, to use a different version change the SOLR_URL value with the preferred release version in the SOLR Dockerfile. Releases are available in https://archive.apache.org/dist/lucene/solr/.
```
Comment (Luukyb) [inline README.md]: Based on your screenshot it looks like 7.6.0 is running. Please confirm.

File: docker/php/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ RUN apt-get update -qq \
   && apt-get install wget mysql-client -yq
 
 # install the PHP extensions we need
-RUN apt-get update && apt-get install -y libpng-dev libjpeg-dev libpq-dev \
+RUN apt-get update && apt-get install -y libpng-dev libjpeg-dev libpq-dev libzip-dev \
```
Comment (Luukyb) [inline docker/php/Dockerfile:7]: You had the same issue as Issue #64 ?

Comment (Luukyb) [reply docker/php/Dockerfile:7]: How about in the Apache branch?

**Comment by mtem27** [comment]
Hi @Luukyb ,

This branch is so messy now, I would like to request for this PR to be closed.

I have created a new one, in https://github.com/expresso-php/expresso-php/pull/66.

Please check there.

Thanks.

Tere

**Comment by Luukyb** [comment]
Noted, thanks @mtem27

**Pull request closed without merging**